src/components/App.js

```diff
@@ -220,6 +224,23 @@ export default class App extends React.Component {
                 />
             </div>
           </section>
+          <section>
+            <AssetsChart />
+            <AccountsChart />
+            <OperationsChart />
+            <TradeVolumeChart assetPair="XLM_BTC" />
+            <TradeVolumeChart assetPair="XLM_CNY" />
+            <TradeVolumeChart assetPair="XLM_EUR" />
+            <TradeVolumeChart assetPair="XLM_JPY" />
+            <TradeVolumeChart assetPair="XLM_PHP" />
+            <TradeVolumeChart assetPair="XLM_XRP" />
+            <TradeVolumeChart assetPair="XLM_GFT" />
+            <TradeVolumeChart assetPair="XLM_SEED" />
+            <TradeVolumeChart assetPair="XLM_XEL" />
+            <TradeVolumeChart assetPair="XLM_LTC" />
+            <TradeVolumeChart assetPair="XLM_EQD" />
+            <TradeVolumeChart assetPair="XLM_ICN" />
+          </section>
```

Comment:
It's missing a margin:
![screen shot 2017-09-20 at 21 12 21](https://user-images.githubusercontent.com/464938/30662734-71fa461e-9e48-11e7-8060-9b803c988a35.png)